Fortnox.NET/Models/Order/Order.cs

```diff
@@ -138,12 +138,12 @@ public class Order
         public string TermsOfPayment { get; set; }
 
         [JsonReadOnly]
-        public float Total { get; set; }
+        public float? Total { get; set; }
 
         [JsonReadOnly]
-        public float TotalVAT { get; set; }
+        public float? TotalVAT { get; set; }
 
-        public bool VATIncluded { get; set; }
+        public bool? VATIncluded { get; set; }
```

Comment:
There seems to be some changes not related to the title of the PR - intentional?

Reply:
I didn't add it in the title but in the description:  "I also made changes to the Order tests and Asset tests to make them run green again."

Theese are intentional changes.

Reply:
ok great